integration.cy.js

```diff
@@ -62,7 +62,7 @@ beforeEach(() => {
 describe("Smoke Test", () => {
    it("App Loads", () => {
       cy.visit("/");
-      cy.get('[data-cy="portal_work_menu_sidebar"]')
+      cy.get('[data-cy="portal_work_menu_sidebar"]', { timeout: 30000 })
```

Comment:
I think this would be better palced in a config file rather than in the tests. It should be `defaultCommandTimeout` in the config, or your can set the environment variable `CYPRESS_DEFAULT_COMMAND_TIMEOUT`.
FYI: we use a `30000` timeout in our github ci runs.